actors/miner/src/expiration_queue.rs

```diff
@@ -715,8 +715,6 @@ impl<'db, BS: Blockstore> ExpirationQueue<'db, BS> {
             let keep_going = f(e.try_into()?, expiration_set)?;
 
             if expiration_set.is_empty() {
-                // Mark expiration set as unchanged, it will be removed after the iteration.
```

Comment:
This feature was removed from the AMT and shouldn't be necessary anyways. IIRC, this was designed to preserve go specs-actors behavior before we started compiling to wasm.